auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/VirtualGroupService.java

```diff
@@ -59,6 +59,19 @@ public void cleanupVirtualGroups(String accountId, String environmentCrn) {
         }
     }
 
+    public void assignEnvironmentAdminRole(String userCrn, String environmentCrn) {
+        try {
+            grpcUmsClient.assignResourceRole(userCrn, environmentCrn, grpcUmsClient.getBuiltInEnvironmentAdminResourceRoleCrn(), MDCUtils.getRequestId());
+            LOGGER.debug("EnvironmentAdmin role of {} environemnt is assigned to the {} user", environmentCrn, userCrn);
+        } catch (StatusRuntimeException ex) {
+            if (Code.ALREADY_EXISTS.equals(ex.getStatus().getCode())) {
+                LOGGER.debug("EnvironmentAdmin role of {} environemnt is already assigned to the {} user", environmentCrn, userCrn);
+            } else {
+                throw ex;
```

Comment:
I don't think that we should make the provisioning fail here. Simply logging the error would do.